airflow/providers/apache/pinot/hooks/pinot.py

```diff
@@ -275,7 +275,7 @@ def get_uri(self) -> str:
         endpoint = conn.extra_dejson.get('endpoint', 'query/sql')
         return f'{conn_type}://{host}/{endpoint}'
 
-    def get_records(self, sql: str, parameters: Optional[Union[Iterable, Mapping]] = None) -> Any:
+    def get_records(self, sql: Union[str, List[str]], parameters: Optional[Any] = None, **kwargs) -> Any:
```

Comment:
I spent a lot of time trying to find out the type of `parameters` parameter.
In OracleHook.callproc() we have the `len(parameters)` situation, therefore it type is `parameters: Optional[Union[List, Dict]]`.
But for the other cases, this parameter is passing to many `.execute()` methods and it's hard to figure out what types these libraries allow.

For example in SQLAlchemy in a few places, there is the following comment.
```
:param parameters: Dictionary, tuple, or list of parameters being
         passed to the ``execute()`` or ``executemany()`` method of the
         DBAPI ``cursor``
```
Based on this information I decided to generalize it to `parameters: Optional[Union[Iterable, Mapping]]`. Probably, it might be `Union[Sequence, Mapping]`

Reply:
Ok. That makes sense. I struggled a bit with that too